wikichange/src/content/content.js

```diff
@@ -311,35 +293,6 @@ const renderPopup = () => {
     });
 };
 
-/**
- * Render a simple JS loader by the highlight button
- */
-const renderLoader = () => {
-    let button = document.getElementById("highlightButton");
-    button.disabled = true;
-
-    let loader = document.getElementById("loader");
-    loader.style.paddingTop = "3px";
-    loader.style.border = "5px solid #f3f3f3";
-    loader.style.borderTop = "5px solid #3498db";
-    loader.style.borderRadius = "50%";
-    loader.style.width = "15px";
-    loader.style.height = "15px";
-    loader.style.position = "absolute";
-    loader.style.marginTop = "6.5px";
-    loader.style.marginLeft = "-30px";
-    loader.style.display = "inline-block";
-    loader.style.animation = "spin 2s linear infinite";
-
-    let keyframes = `@keyframes spin {
-        0% { transform: rotate(0deg); }
-        100% { transform: rotate(360deg); }
-    }`;
-    let style = document.createElement("style");
-    style.innerHTML = keyframes;
-    document.head.appendChild(style);
-};
-
```

Comment:
you moved the loader to the css file, but I think the animation (keyframe spin) that makes it spins is missing

Reply:
you keep this if there is not a huge reason to move all these to .css file